spec/views/home/index.html.erb_spec.rb

```diff
@@ -0,0 +1,89 @@
+require 'spec_helper'
+ 	
+describe "home/index.html.erb" do
+
+  context "when not logged in" do
+  
+    before(:each) do
+      view.stub(:logged_in?).and_return(false)
+      view.stub(:logged_in_as_admin?).and_return(false)
+      view.stub(:current_user).and_return(nil)
+    end
+    
+    context "when account creation and invites are disabled" do
+      before(:each) do
+        @admin_settings = mock(Object)
+        @admin_settings.stub(:account_creation_enabled?).and_return(false)
+        @admin_settings.stub(:invite_from_queue_enabled?).and_return(false)
+      end
+      
+      it "should display the typical boxes" do
+        render
+        rendered.should have_selector('.primary.navigation')
+        rendered.should have_selector('.intro.module') # Welcome
+        rendered.should have_selector('.latest.module')
+      end
+
+     it "should not display invite links" do
+        render
+        rendered.should_not contain "Invite a friend"
+        rendered.should_not contain "Get an Invite"
+      end
```

Comment:
`end` at 31, 6 is not aligned with `it "should not display invite links" do` at 27, 5.